packages/flutter_tools/templates/app/windows.tmpl/runner/utils.cpp

```diff
@@ -20,3 +20,41 @@ void CreateAndAttachConsole() {
     FlutterDesktopResyncOutputStreams();
   }
 }
+
+std::vector<std::string> GetCommandLineArguments() {
+  // Convert the UTF-16 command line arguments to UTF-8 for the Engine to use.
+  int argc;
+  wchar_t** argv = ::CommandLineToArgvW(::GetCommandLineW(), &argc);
```

Comment:
Nit: null-check this to make sure it didn't fail; the docs don't actually guarantee that argc will be set to 0 on failure unless I missed it,